xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -121,29 +121,38 @@ void CGUIDialogPVRTimerSettings::SetTimer(const CPVRTimerInfoTagPtr &timer)
   }
 
   // Copy data we need from tag. Do not modify the tag itself until Save()!
-  m_timerType     = m_timerInfoTag->GetTimerType();
+  m_timerType = m_timerInfoTag->GetTimerType();
+
+  if (!m_timerType)
```

Comment:
Are there really any situations where a timer tag could be created and stored without a timer type?